src/pages/TeamPage/TeamPage.js

```diff
@@ -1,14 +1,90 @@
-import React, { useContext } from 'react';
+import React, { useState, useContext } from 'react';
 import { UserContext } from './../../context/UserContext';
 import Menu from '../../components/Menu/Menu';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Snackbar from '@material-ui/core/Snackbar';
+import Alert from '@material-ui/lab/Alert';
+import { useStyles } from './TeamPage.styles';
+import * as validator from '../../helpers/inputValidator';
+import axios from 'axios';
+
 
 const TeamPage = () => {
   const [user, setUser] = useContext(UserContext);
-  console.log(user);
+  const [invitationDialog, setInvitationDialog] = useState(false);
+  const [email, setEmail] = useState({ input: null, error: false, helperText: null });
+  const [alert, setAlert] = useState({ open: false, message: null, severity: null });
+
+  const classes = useStyles();
+
+  const handleInviationDialogOpen = () => {
```

Comment:
I would recommend not to create separate methods here, because you can use just setInvitationDialog(true); instead. You don't need separate function for that.

Reply:
Good point